pkg/tarutil/tar.go

```diff
@@ -26,6 +27,10 @@ type Opts struct {
 	// to include all sub-directories. Set to true to prevent this
 	// behavior.
 	NoRecursion bool
+
+	// Change to this directory before any operations. Generated tar file
+	// records use the original filename.
+	ChangeDirectory string
```

Comment:
Practically, do you foresee this would be set to a different value apart from root (`/`) in any case ? if so, what case would it be ?

Reply:
Take a look at these answers on stack overflow for a common use case of `-C`: https://stackoverflow.com/questions/18681595/tar-a-directory-but-dont-store-full-absolute-paths-in-the-archive

Reply:
(srry, maybe I am mis-reading) but looks like current impl here assumes `ChangeDirectory` is always `/`, for example

(line 141~ 144)
```
			if filepath.IsAbs(bFile) {
				// "cd" does nothing if the file is absolute.
				bcPath = abcPath
			}
```

Reply:
This was moved to #2213. There is no longer a `-C` flag. Just the `ChangeDirectory` option.